examples/gopherlouse.mapping

```diff
@@ -0,0 +1,10 @@
+p26:h10
```

Comment:
This file was renamed to `gopher_louse_map.mapping`. Please remove this file use `gopher_louse_map.mapping` instead. Alternatively, you can rename `gopher_louse_map.mapping` to `gopher_louse.mapping` and remove `gopher_louse_map.mapping`. If you choose to do this, you also have to modify tests that use `gopher_louse_map.mapping`).

Reply:
Removed.